packages/mobile/src/store/store.ts

```diff
@@ -41,7 +41,7 @@ export type AppState = {
   oauth: OAuthState
   remoteConfig: RemoteConfigState
   search: SearchState
-  signon: SignonState
+  signonDeprecated: SignonState
```

Comment:
nice idea, lets just use this pattern going forward?

Reply:
renamed legacy